packages/radix-ui-themes/src/styles/tokens/color.css

```diff
@@ -172,11 +172,12 @@
 
   /* Because Chrome is buggy with box-shadow transitions from "transparent" keyword and/or RGB color into P3 colors */
   --color-transparent: rgb(0 0 0 / 0);
-  @supports (color: color(display-p3 1 1 1)) {
-    @media (color-gamut: p3) {
-      & {
-        --color-transparent: color(display-p3 0 0 0 / 0);
-      }
+}
+
+@supports (color: color(display-p3 1 1 1)) {
+  @media (color-gamut: p3) {
+    .radix-themes {
+      --color-transparent: color(display-p3 0 0 0 / 0);
```

Comment:
tbh I expected exactly this result from postcss-nested, but somehow it changes everything

Reply:
Alternatively, we can change all `.radix-themes` rules which are paired with dark mode rules into `:where(.radix-themes)` to guarantee that dark mode takes over

Reply:
I'll actually do this